ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-master.yaml

```diff
@@ -50,17 +49,19 @@ resources:
 tag_specification:
   name: "4.8"
   namespace: ocp
+test_binary_build_commands: go mod vendor
```

Comment:
The no go files error is usually a permission issue in prow; So we probably need to add this here in the test bin command:

`mkdir /.cache && chmod 775 -R /.cache && chmod 775 -R $GOPATH`